tests/parametric/test_otel_span_methods.py

```diff
@@ -61,12 +83,29 @@ def test_otel_set_service_name(self, test_agent, test_library):
         assert root_span["name"] == "parent_span"
         assert root_span["service"] == "new_service"
 
-    @irrelevant(context.library == "cpp", reason="library does not implement OpenTelemetry")
+    @missing_feature(context.library == "golang", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "java", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "nodejs", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "dotnet", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "python", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "python_http", reason="New operation name mapping not yet implemented")
+    def test_otel_set_service_name(self, test_agent, test_library):
+        """
+            - Update the service name on a span
+        """
+        with test_library:
+            with test_library.otel_start_span("parent_span", span_kind=SK_INTERNAL) as parent:
+                parent.set_attributes({"service.name": "new_service"})
+                parent.end_span()
+
+        root_span = get_span(test_agent)
+        assert root_span["name"] == "internal"
+        assert root_span["resource"] == "parent_span"
+        assert root_span["service"] == "new_service"
+
     @irrelevant(context.library >= "java@1.22.0", reason="library implements the new array encoding")
     @missing_feature(context.library == "nodejs", reason="Empty string attribute value are not supported")
-    @missing_feature(context.library == "php", reason="Not implemented")
-    @missing_feature(context.library == "ruby", reason="Not implemented")
-    def test_otel_set_attributes_different_types(self, test_agent, test_library):
+    def test_otel_set_attributes_different_types_old_name(self, test_agent, test_library):
```

Comment:
Is this going to be deleted then, since it now has "old" in the test name? It looks very similar to the following test.

Reply:
Deleted all the old ones in favor of the new ones in 4cb9bb49946547e19f35485a611bea6e56a23da2
I need to unskip them though still